system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
+Documentation=man:netdata
+Documentation=file:///usr/share/doc/netdata/html/index.html
+Documentation=https://github.com/firehol/netdata
 After=network.target httpd.service squid.service nfs-server.service mysqld.service named.service postfix.service
+Wants=network-online.target
+ConditionPathExists=/etc/netdata/netdata.conf
 
 [Service]
 Type=simple
+Environment="netdata_LOG_LOCATION=/var/log/netdata/log"
+ExecStart=@sbindir_POST@/netdata -D
+ExecReload=@sbindir_POST@/netdata reload
+TimeoutStopSec=60
+KillMode=mixed
+KillSignal=SIGTERM
+OOMScoreAdjust=-900
+
 User=netdata
 Group=netdata
-ExecStart=@sbindir_POST@/netdata -D
+Restart=on-abnormal
+RestartSec=2s
+LimitNOFILE=65536
 
-# saving a big db on slow disks may need some time
-TimeoutStopSec=60
+WorkingDirectory=/tmp
+
+# Hardening
+
+NoNewPrivileges=false
+PermissionsStartOnly=true
```

Comment:
PermissionsStartOnly: usually other Exec* commands, were used, need permissions different than the daemon runtime.

NoNewPrivileges is set to false otherwise it would break apps.plugin - however it would be good to be able to set it to true and give apps.plugin the right capabilities. (See below)